pybamm/meshes/scikit_fem_submeshes.py

```diff
@@ -0,0 +1,139 @@
+#
+# scikit-fem meshes for use in PyBaMM
+#
+import pybamm
+
+import numpy as np
+import importlib
+
+skfem_spec = importlib.util.find_spec("skfem")
+if skfem_spec is not None:
+    skfem = importlib.util.module_from_spec(skfem_spec)
+    skfem_spec.loader.exec_module(skfem)
+
```

Comment:
a bit neater to use something like `have_scikits_fem` similar to `have_scikits_odes` (see `scikits_ode_solver.py`). Though given that it's just a pip package, we can probably assume it will always be installed anyway